packages/ai-openai/src/browser/openai-frontend-application-contribution.ts

```diff
@@ -38,72 +40,119 @@ export class OpenAiFrontendApplicationContribution implements FrontendApplicatio
             this.manager.setApiKey(apiKey);
 
             const models = this.preferenceService.get<string[]>(MODELS_PREF, []);
-            this.manager.createOrUpdateLanguageModels(...models.map(createOpenAIModelDescription));
+            const requestSettings = this.preferenceService.get<RequestSetting[]>(PREFERENCE_NAME_REQUEST_SETTINGS, []);
+            this.manager.createOrUpdateLanguageModels(...models.map(modelId => this.createOpenAIModelDescription(modelId, requestSettings)));
             this.prevModels = [...models];
 
             const customModels = this.preferenceService.get<Partial<OpenAiModelDescription>[]>(CUSTOM_ENDPOINTS_PREF, []);
```

Comment:
This type is no longer correct as it was extended by the `defaultRequestSettings` which are in fact not stored in this preference.